### ISSUE TITLE
mxnet.ndarray.sparse.CSRNDArray.astype docs error

### ISSUE THREAD

**Issue by aaronmarkham** [apache/incubator-mxnet#13025]
## Description
Sphinx throws an error when generating the docs for this function.

## Error
```
/home/ubuntu/incubator-mxnet/python/mxnet/ndarray/sparse.py:docstring of mxnet.ndarray.sparse.CSRNDArray.astype:5: ERROR: Unexpected indentation.
/home/ubuntu/incubator-mxnet/python/mxnet/ndarray/sparse.py:docstring of mxnet.ndarray.sparse.CSRNDArray.astype:8: WARNING: Block quote ends without a blank line; unexpected unindent.
```

**Comment by frankfliu** [comment]
@mxnet-label-bot [Doc]

**Comment by vrakesh** [comment]
PR merged and issue fixed, issue good to close

**Comment by sad-** [comment]
@aaronmarkham can we close out this issue now. pr looks to be merged

**Comment by aaronmarkham** [comment]
Fixed with #13171
